auto/src/glewinfo_tail.c

```diff
@@ -11,35 +11,44 @@ int main (void)
 #endif
 {
   GLuint err;
+  struct createParams params = {
+#if defined(_WIN32)
+    -1,  /* pixelformat */
+#elif !defined(__APPLE__) && !defined(__HAIKU__) || defined(GLEW_APPLE_GLX)
+    "",  /* display */
+    -1,  /* visual */
+#endif
+    0,   /* major */
+    0,   /* minor */
+    0,   /* profile mask */
+    0    /* flags */
+  };
 
 #if defined(_WIN32) || !defined(__APPLE__) || defined(GLEW_APPLE_GLX)
-  char* display = NULL;
-  int visual = -1;
-
-  if (glewParseArgs(argc-1, argv+1, &display, &visual))
+  if (glewParseArgs(argc-1, argv+1, &params))
   {
+    fprintf(stderr, "Usage: glewinfo "
 #if defined(_WIN32)
-    fprintf(stderr, "Usage: glewinfo [-pf <id>]\n");
+	    "[-pf <pixelformat>] "
 #else
-    fprintf(stderr, "Usage: glewinfo [-display <display>] [-visual <id>]\n");
+	    "[-display <display>] "
+	    "[-visual <visual id>] "
 #endif
+	    "[-version <OpenGL version>] "
+	    "[-profiles <OpenGL profile mask>] "
+	    "[-flags <OpenGL flags>]"
+	    "\n");
     return 1;
   }
 #endif
 
-#if defined(_WIN32)
-  if (GL_TRUE == glewCreateContext(&visual))
-#elif defined(__APPLE__) && !defined(GLEW_APPLE_GLX)
-  if (GL_TRUE == glewCreateContext())
-#else
-  if (GL_TRUE == glewCreateContext(display, &visual))
-#endif
+  if (GL_TRUE == glewCreateContext(&params))
   {
     fprintf(stderr, "Error: glewCreateContext failed\n");
     glewDestroyContext();
     return 1;
   }
-  glewExperimental = GL_TRUE;
+  //glewExperimental = GL_TRUE;
```

Comment:
I changed that for testing and shouldn't have commited it, I will revert the change.